pkg/plugin/processor/builtin/impl/filter.go

```diff
@@ -35,10 +35,12 @@ func (p *filterProcessor) Specification() (sdk.Specification, error) {
 	return sdk.Specification{
 		Name:    "filter",
 		Summary: "Acknowledges all records that get passed to the filter.",
-		Description: `Acknowledges all records that get passed to the filter, so the records will be filtered out if 
-the condition provided to the processor is evaluated to "true".
+		Description: `Acknowledges all records that get passed to the filter, so
+the records will be filtered out if the condition provided to the processor is
+evaluated to ` + "`true`" + `.
 
-**Important:** Make sure to add a condition to this processor, otherwise all records will be filtered out.`,
+**Important:** Make sure to add a [condition](https://conduit.io/docs/processors/conditions)
```

Comment:
I see this page is still not done, so I left a comment in https://github.com/ConduitIO/conduit/issues/1422 to not forget url is this one.

Reply:
Good catch, thanks!